python/lsst/analysis/drp/skyPlot.py

```diff
@@ -233,14 +264,18 @@ def skyPlot(self, catPlot, plotInfo, sumStats):
         zCol = self.config.axisLabels["z"]
 
         # For galaxies
-        xsGalaxies = catPlot.loc[galaxies, xCol]
-        ysGalaxies = catPlot.loc[galaxies, yCol]
-        colorValsGalaxies = catPlot.loc[galaxies, zCol]
+        colorValsGalaxies = catPlot.loc[galaxies, zCol].values
+        ids = extremaSort(colorValsGalaxies)
+        xsGalaxies = catPlot.loc[galaxies, xCol].values[ids]
+        ysGalaxies = catPlot.loc[galaxies, yCol].values[ids]
+        colorValsGalaxies = colorValsGalaxies[ids]
 
         # For stars
-        xsStars = catPlot.loc[stars, xCol]
-        ysStars = catPlot.loc[stars, yCol]
-        colorValsStars = catPlot.loc[stars, zCol]
+        colorValsStars = catPlot.loc[stars, zCol].values
+        ids = extremaSort(colorValsStars)
+        xsStars = catPlot.loc[stars, xCol].values[ids]
+        ysStars = catPlot.loc[stars, yCol].values[ids]
+        colorValsStars = colorValsStars[ids]
 
         # Calculate some statistics
         if np.any(catPlot["sourceType"] == 2):
```

Comment:
The `ax.text` for source type == 2 (galaxies) is:
```
ax.text(0.63, 0.91, ...
```
however, for all other types (stars/unknown) it is:
```
ax.text(0.8, 0.91, ...
```
These should be reconciled to be the same (i.e., changing the galaxies value to 0.63).

Reply:
This was here so that if stars and galaxies were plotted on the same plot both could be seen. I was thinking that I would change these to all be the same but then people have mentioned maybe wanting to plot both so I think I will leave it.

Reply:
Actually I made it figure out if two types were being plotted and only move it over if needed.